usr/src/test/zfs-tests/cmd/scripts/zfstest.ksh

```diff
@@ -136,10 +136,10 @@ function constrain_path
 	# SmartOS does not ship some required commands by default.
 	# Link to them in the package manager's namespace.
 	pkgsrc_bin=/opt/tools/bin
-	pkgsrc_packages="sudo truncate python base64 shuf sha256sum"
+	pkgsrc_packages="fio uuidgen md5sum sudo truncate python base64 shuf sha256sum"
```

Comment:
I thought in the illumos test suites md5sum was a shell function that invoked "digest -a md5"?  (If it doesn't it should, same with sha256sum...)

Probably not a dealbreaker.

Reply:
hm, well, of course digest would make sense. The md5sum is used by:
20240802T212851/log:22:32:25.61 /opt/zfs-tests/tests/functional/cli_root/zpool_import/import_rewind_config_changed[220]: test_add_vdevs[156]: test_common[133]: log_must[67]: log_pos[253]: verify_data_md5sums[144]: md5sum: not found [No such file or directory]
20240802T212851/log:22:32:25.61 ERROR: verify_data_md5sums /var/tmp/md5sums.75391 exited 127

well, digest is currently used directly too, by different test scripts, except that indeed, there is a sha256digest function in libtest.shlib. I think, we probably want to address this in separate issue, or rather directly in upstream.